app.py

```diff
@@ -1,25 +1,49 @@
 import connexion
```

Comment:
🏕️ Could this file be added to some connexion template repo? Just a thought.

Reply:
To discuss?